src/main/java/org/example/OccurrenceWordPoc.java

```diff
@@ -1,29 +1,40 @@
 package org.example;
 
 import java.io.File;
+import java.sql.SQLException;
 
+/**
+ * In this Poc here the inputs are received from the user, then shows a  message ‘Processing’
+ * then the thread will read the file contents, and search for the word given.
+ * counts the number of occurrences of the word and pass the info back to the main thread.
+ * and then Handles the error,then all application is logged to mysql database and table is created
+ * with specified contents.
+ */
 public class OccurrenceWordPoc {
     /*
     Entering the path
     once if the input is received it shows processing
     and executes if the file is present or not present
      */
-    public static void main(String[] args) {
-        if (args.length == 2) {
+    public static void main(String[] args) throws SQLException {
+        if (args.length == Constants.ArgumentNumber) {
             String inputFilePath = args[0];
             String searchWord = args[1];
             System.out.println(inputFilePath);
             if (inputFilePath.endsWith(".txt") || inputFilePath.endsWith(".json")) {
```

Comment:
Include the .txt and .json and assgin them to variables like txtExtension and jsonExtension in Constants class and access them from that class.

Reply:
I have Updated has Suggested.